data/json/items/armor/combat_exoskeleton.json

```diff
@@ -11,7 +11,7 @@
     "weight": "13000 g",
     "environmental_protection": 20,
     "warmth": 40,
-    "flags": [ "STURDY", "PADDED", "WATERPROOF", "RAINPROOF", "RAD_PROOF", "ELECTRIC_IMMUNE", "SUN_GLASSES", "COMBAT_TOGGLEABLE" ],
+    "flags": [ "STURDY", "PADDED", "WATERPROOF", "RAINPROOF", "RAD_PROOF", "ELECTRIC_IMMUNE", "SUN_GLASSES", "COMBAT_TOGGLEABLE", "MODULE_HOLDER" ],
```

Comment:
Not sure what it is complaining about.

Reply:
Sort of. You will want to run your changes through a JSON formatter. When a line gets too long, in most cases, it's preferred to break it up into smaller chunks, so that the parent item is shorter per line (which works well with arrays, for example).

Reply:
Thanks for the insight!